### ISSUE TITLE
Ability to exclude some paths (not deps) from checking

### ISSUE THREAD

**Issue by anentropic** [fredrikaverpil/creosote#184]
### What feature request would you like to suggest?

I have a Django project

Unfortunately in the static files there is a JS package that has a Python 2 script in it, so when I run creosote I get:
```
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/static/assets/vendors/ionicons-npm/builder/generate.py", line 29
    print "Generate Fonts"
    ^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
```

I have a file structure like:
```
app/
  a/
    ...py files to check...
  b/
    ...py files to check...
  c/
    ...py files to check...
  static/
    assets/
      ...somewhere in here a bad JS lib...
  t/
    ...py files to check...
  u/
    ...py files to check...
core/
    ...py files to check...
scripts/
    ...py files to check...
tests/
    ...py files to check...
```

so I am running creosote like:
```
creosote --p scripts -p core -p tests -p apps --verbose --use-feature v3-args
```

it looks like my only option is instead of `-p apps` to manually specify all the relevant subdirs of apps/ as a dozen more `-p` args

so I'd love to have a flag that is the opposite of `-p` to exclude dirs

### Will the feature request break backwards compatibility?

not if it's just adding a new cli arg

**Comment by fredrikaverpil** [comment]
Hi @anentropic and thanks for the suggestion. I likely won't be doing this but you are free to take a stab at this as I'm accepting PRs.

You should be able to add a new argument that kan take multiple arguments, such as `--paths` or `--venvs`. Then you can modify the code [here](https://github.com/fredrikaverpil/creosote/blob/a47ba81845a22ea0c97503459f97c05767b92513/src/creosote/parsers.py#L218), so to make creosote not parse those files.

I would recommend you do this on top of the work currently being done in #183 as this will also make the `v3-args` the default and will get released as a 3.0.0 version.

**Comment by fredrikaverpil** [comment]
@anentropic you should now be able to quite easily add a test for what you want to do in `test_integration.py`, add a new commandline argument in `config.py` finally take advantage of the new argument's value [here](https://github.com/fredrikaverpil/creosote/blob/a47ba81845a22ea0c97503459f97c05767b92513/src/creosote/parsers.py#L218).

I've made this PR "hacktober-eligible". 😄
